pkg/distributor/distributor_test.go

```diff
@@ -3498,40 +3630,39 @@ func prepare(t testing.TB, cfg prepConfig) ([]*Distributor, []mockIngester, []*p
 			timeOut:                       cfg.timeOut,
 		})
 	}
-	for i := cfg.happyIngesters; i < cfg.numIngesters; i++ {
-		ingesters = append(ingesters, mockIngester{
-			queryDelay:       cfg.queryDelay,
-			pushDelay:        cfg.pushDelay,
-			seriesCountTotal: cfg.ingestersSeriesCountTotal,
-		})
-	}
+	return ingesters
+}
 
-	// Use a real ring with a mock KV store to test ring RF logic.
+func prepareRingInstances(ingesters []mockIngester) *ring.Desc {
 	ingesterDescs := map[string]ring.InstanceDesc{}
-	ingestersByAddr := map[string]*mockIngester{}
+
 	for i := range ingesters {
-		addr := fmt.Sprintf("%d", i)
```

Comment:
the address was opaque, so changing it to something more involved shouldn't make a difference